CODE_REVIEW_15.md

```
@@ -0,0 +1,357 @@
+    var cmp int
+    switch sortColumn {
+    case projectSortName:
+        // ...
+    case projectSortCPU:
+        // ...
+    // Pas de default case
+    }
+```
+
+**Suggestion:** Ajouter un default case défensif :
+```go
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)